core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/RenameConnector.java

```diff
@@ -110,7 +111,9 @@ public Response rename(@QueryParam("oldPath") String oldPath,
       }
       String newExoTitle = newTitle;
       // Clarify new name & check to add extension
-      String newName = Text.escapeIllegalJcrChars(Utils.cleanString(newTitle));
+      String newName  = Text.escapeIllegalJcrChars(newTitle);
```

Comment:
`Text.escapeIllegalJcrChars` should be sufficient to remove accent and other characters.

Reply:
ok